app/models/user.rb

```diff
@@ -2,6 +2,8 @@ class User < ApplicationRecord
   has_secure_password
   attr_accessor :remember_token
 
+  scope :newest, ->{order(created_at: :desc)}
```

Comment:
```suggestion
  scope :newest, -> {order(created_at: :desc)}
```
Thêm space trống trước block e nha, setup lại rubocop chỗ này